content/docs/operating/storage.md

```diff
@@ -86,3 +86,12 @@ storage directory:
    1. Stop Prometheus.
    1. `rm -r <storage path>/*`
    1. Start Prometheus.
+
+## Hash collisions
+
+Prometheus uses 64-bit fingerprints to identify time series. On a
```

Comment:
Yeah, but then the details depend a lot on the hash quality (FNV-1a). Someone did some collision and randomness comparisons here: http://programmers.stackexchange.com/questions/49550/which-hashing-algorithm-is-best-for-uniqueness-and-speed

But it's only for the 32-bit versions... but at least FNV-1a seems to have reasonably good (but *really* good) randomness.